homeassistant/components/ezviz/alarm_control_panel.py

```diff
@@ -0,0 +1,133 @@
+"""Support for Ezviz alarm."""
+import logging
+from typing import Any
+
+from pyezviz.constants import DefenseModeType
+from pyezviz.exceptions import HTTPError
+
+from homeassistant.components.alarm_control_panel import AlarmControlPanelEntity
+from homeassistant.components.alarm_control_panel.const import (
+    SUPPORT_ALARM_ARM_AWAY,
+    SUPPORT_ALARM_ARM_NIGHT,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    STATE_ALARM_ARMED_AWAY,
+    STATE_ALARM_ARMED_NIGHT,
+    STATE_ALARM_DISARMED,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.restore_state import RestoreEntity
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import (
+    ATTR_AWAY,
+    ATTR_HOME,
+    ATTR_SLEEP,
+    DATA_COORDINATOR,
+    DOMAIN,
+    MANUFACTURER,
+)
+from .coordinator import EzvizDataUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up Ezviz alarm control panel."""
+    coordinator: EzvizDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id][
+        DATA_COORDINATOR
+    ]
+
+    async_add_entities([EzvizAlarm(coordinator)])
+
+
+class EzvizAlarm(CoordinatorEntity, AlarmControlPanelEntity, RestoreEntity):
```

Comment:
All your static properties and instance attributes should be defined as class attributes

Reply:
Done. Moved all the static instance properties to class attributes.